federatedscope/core/auxiliaries/data_builder.py

```diff
@@ -611,3 +611,17 @@ def merge_data(all_data):
                 merged_data[d_name][elem_name])
 
     return merged_data
+
+
+def merge_test_data(all_data):
```

Comment:
please provide docstring for this method.

Reply:
Done